igor-web/src/main/groovy/com/netflix/spinnaker/igor/gitlabci/service/GitlabCiService.java

```diff
@@ -175,12 +177,12 @@ private Map<String, Object> getPropertyFileFromLog(String projectId, Integer pip
 
             return properties;
 
-          } catch (RetrofitError e) {
+          } catch (SpinnakerServerException e) {
             // retry on network issue, 404 and 5XX
```

Comment:
SpinnakerNetworkException is already retryable, so I think it's sufficient to catch SpinnakerHttpException here.  It seems a little strange to retry on 404, but so be it.